formula_renames.json

```diff
@@ -131,5 +131,10 @@
   "v8-315": "v8@3.15",
   "wires": "geckodriver",
   "wpcli-completion": "wp-cli-completion",
-  "xu4": "scummvm"
+  "xu4": "scummvm",
+  "kde-extra-cmake-modules": "extra-cmake-modules",
+  "kde-karchive": "karchive",
+  "kde-ki18n": "ki18n",
+  "kde-kdoctools": "kdoctools",
+  "kde-threadweaver": "threadweaver"
```

Comment:
The rest of this list is sorted alphabetically, can we stick to that?

Reply:
Don't know how I missed that. Fixed.